protocol/persistence.go

```diff
@@ -755,6 +755,22 @@ func (db sqlitePersistence) SaveContactChatIdentity(contactID string, chatIdenti
 		_ = tx.Rollback()
 	}()
 
+	if len(chatIdentity.Images) == 0 {
```

Comment:
I am wondering if this is a suitable approach. In the future, we might want to remove only images of a specific type. In this scenario, an image will be removed only if there are no other images present at all.

The solution could be:
- collect all image types that are left
- then `DELETE FROM chat_identity_contacts WHERE contact_id = ? AND image_type NOT IN (?, ?, ...)`, `tx.Exec(contactID, imageTypes)`

Reply:
Hey, thanks for your comment @osmaczko. I'll be reviewing the code with this new perspective. Seems to me like an amazing comment on my unknown unknowns 🙏🏾

Reply:
It turns out, the keys to the chatIdentity.Images is actually a string that represents the type of images. This type could be one of :

```
SmallDimName = "thumbnail"
LargeDimName = "large"
BannerIdentityName = "banner"
```

The idea is to delete the image matching only the image type above.

Reply:
I have updated the code to handle the removing of images of specific type. I agree that this udpate makes the code more robust for future possible improvements like stated above, for instance when deleting 1 of 3 images.
I have also renamed the confusing `ImageType` to `ImageFormat`
I believe those changes should be enough to complete this status-go PR. Sending another revision.
Thanks!